### DISCUSSION TITLE
Welcome to transdim Discussions!

### DISCUSSION THREAD

**Discussion by xinychen** [xinychen/transdim#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Junxi-Man** [comment]
Hi, I'm really interested in your datasets! Could you share the origin of the NGSIM dataset? I'd love to learn more about it. Thanks a lot!
